db/migrate/20170721181533_add_deleted_at_to_tables.rb

```diff
@@ -0,0 +1,18 @@
+class AddDeletedAtToTables < ActiveRecord::Migration[5.1]
+  def up
+    add_column(:users, :deleted_at, :datetime)
+    add_index(:users, :deleted_at)
+
+    add_column(:websites, :deleted_at, :datetime)
+    add_index(:websites, :deleted_at)
+
+    add_column(:pings, :deleted_at, :datetime)
+    add_index(:pings, :deleted_at)
+  end
+
+  def down
+    remove_column(:users, :deleted_at)
```

Comment:
When you remove the column, the index is automatically removed as well. I only added `up`/`down` to prevent PostgreSQL from throwing an error trying to remove an index that doesn't exist.